ccm_web/server/src/migrations/2021.06.30T19.52.46.add-canvas-token-expires-at.ts

```diff
@@ -0,0 +1,19 @@
+import Sequelize, { DataTypes } from 'sequelize'
+import { Migration } from '../migrator'
+
+export const up: Migration = async ({ context: sequelize }) => {
+  await sequelize.getQueryInterface().addColumn(
+    'canvas_token',
+    'expires_at',
```

Comment:
It's kind of unusual to me that, unlike the `CanvasToken` migration, you don't ever actually specify the model's property name in `camelCase` (e.g. `expiresAt`). The mapping still works though, go figure. If you change this first usage to `expiresAt`, it seems to set the database column name to that, even with the `field` specified in options below.